### PR TITLE
adding PyTorch and PyG inference pages

### PR DESCRIPTION
**Pull request by GageDeZoort**

### CODE REVIEW

File: content/inference/pytorch.md [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+gpu time per call: 0.000011 s
+gpu_time/cpu_time: 0.000419
+```
+
+The complete list of Torch Tensor operations is available in the [docs](https://pytorch.org/docs/stable/torch.html?highlight=mm). 
+
+#### Autograd
+
+Backpropagation occurs automatically through autograd. For example, consider the following function and its derivatives:
+\begin{align}
```
Comment (aperloff) [inline content/inference/pytorch.md:146]: Will this be rendered properly once merged? It's not being rendered on the GitHub markdown preview, but maybe it will be by mkdocs.

File: content/inference/pytorch.md [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+### Via ONNX
+One way to incorporate your PyTorch models into CMSSW is through the [Open Neural Network Exchange](https://www.onnxruntime.ai/about.html) (ONNX) Runtime tool. In brief, ONNX supports training and inference for a variety of ML frameworks, and is currently integrated into CMSSW (see the CMS ML tutorial).  PyTorch hosts an excellent tutorial on [exporting a model from PyTorch to ONNX](https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html). ONNX is available in CMSSW (see a relevant [discussion](https://github.com/cms-sw/cmssw/issues/27458) in the CMSSW git repo). 
+
+#### Example Use Cases 
+The ZZ\\(\rightarrow\\)4b analysis utilizes trained PyTorch models via ONNX in CMSSW (see the corresponding [repo](https://github.com/patrickbryant/ZZ4b/blob/master/README.md)). Briefly, they run ONNX in CMSSW_11_X via the CMSSW package `PhysicsTools/ONNXRuntime`, using it to define a [multiClassifierONNX](https://github.com/patrickbryant/ZZ4b/blob/5931a21d8005683e23166c0b44b9594b52ad1126/nTupleAnalysis/interface/multiClassifierONNX.h) class. This multiclassifier is capable of loading pre-trained PyTorch models specified by a `modelFile` string as follows:
```
Comment (aperloff) [inline content/inference/pytorch.md:393]: Same comment about LaTeX rendering.

File: content/inference/pytorch.md [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+
+A Graph Attention Network (GAN) is available via Triton in CMSSW, and can be accessed here: https://github.com/cms-sw/cmssw/tree/master/HeterogeneousCore/SonicTriton/test
+
+## Training Tips
+- When instantiating a `DataLoader`, `shuffle=True` should be enabled for training data but not for validation and testing data. At each training epoch, this will vary the order of data objects in each batch; accordingly, it is not efficient to load the full dataset (in its original ordering) into GPU  memory before training. Instead, enable `num_workers>1`; this allows the `DataLoader` to load batches to the GPU as they're prepared. Note that this launches muliple processerson the CPU. For more information, see a corresponding [discussion](https://discuss.pytorch.org/t/keras-trains-significantly-faster-than-pytorch-for-simple-network/124303/5) in the PyTorch forum.
```
Comment (aperloff) [inline content/inference/pytorch.md:410]: "multiple processerson the" -> "multiple processors on"

Do you mean processors or threads?
